weblogic/datadog_checks/weblogic/data/metrics.yaml

```diff
@@ -0,0 +1,127 @@
+## Default metrics collected by this check. You should not have to modify this.
+jmx_metrics:
+  - include:
+      domain: com.bea
+      bean_regex: com.bea:Name=([-.\w]+),Type=ServerRuntime
+      attribute:
+        OpenSocketsCurrentCount:
+          alias: weblogic.server_runtime.open_sockets_count
```

Comment:
should this be monotonic count?

Reply:
I kept this as gauge because the MBean [reference](https://docs.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/wlmbr/mbeans/ServerRuntimeMBean.html?skipReload=true#OpenSocketsCurrentCount) describes it as:
> The current number of sockets registered for socket muxing on this server.

Which I took to mean that the value may increase or decrease over time. What do you think?

Reply:
That makes sense. The `_count` suffix made me think it was suppose to be a monotonic_count. If it's a gauge, we can probably drop the suffix. But it's up to you, I'm just going off the recent updates in openmetrics V2 where .count means it's a count.

Reply:
Got it, I'll drop the `_count` suffix.

Reply:
Should I do the same for the other gauge metrics with a `_count` suffix?

Reply:
Yes I think consistency is best